dlt/load/utils.py

```diff
@@ -216,3 +218,45 @@ def _extend_tables_with_table_chain(
                 continue
             result.add(chain_table_name)
     return result
+
+
+def filter_new_jobs(
+    file_names: Sequence[str],
+    capabilities: DestinationCapabilitiesContext,
+    config: LoaderConfiguration,
+) -> Sequence[str]:
```

Comment:
This will now use the capabilities of the main destination for all jobs, regardless of the settings on the staging destination and regardless of which jobs are for staging or main destination. I can add this differentiation too, but I'm not sure we really need it..

Reply:
we don't need this now. eventually the more strict settings could be used. We already have function that combines capabilities in Pipeline. we could extend it and use it here. but definitely not in scope of this